examples/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project(ArrayFire-Examples
   VERSION 3.5.0
   LANGUAGES CXX)
 
+if (NOT ASSETS_DIR)
+    set(ASSETS_DIR "" CACHE PATH "Please provide the path to assets for ArrayFire examples")
```

Comment:
The Doc string should describe the variable not instruct or ask for info from the user. Use something like

The assets directory containing images and other data for examples. Download from blah...